.kokoro/BuildGeneratedDocs.sh

```diff
@@ -23,9 +23,16 @@ declare -r supportversion=$(grep \<Version\> Src/Support/CommonProjectProperties
 # TODO: Work out somewhere better to put these!
 rm -rf obj
 mkdir obj
-curl -sSL https://googleapis.dev/dotnet/Google.Apis/$supportversion/xrefmap.yml -o obj/Google.Apis.xrefmap.yml
-curl -sSL https://googleapis.dev/dotnet/Google.Apis.Core/$supportversion/xrefmap.yml -o obj/Google.Apis.Core.xrefmap.yml
-curl -sSL https://googleapis.dev/dotnet/Google.Apis.Auth/$supportversion/xrefmap.yml -o obj/Google.Apis.Auth.xrefmap.yml
+
+# Temporary workaround for failure to fetch xref maps from googleapis.dev: build locally and copy.
+# This is far from great, as it means we could refer to unreleased methods, but it's better than nothing.
```

Comment:
So, now support docs will always be on par with HEAD, right?

I'm fine with that, just to double check that I'm understanding things right.

Reply:
This is only when building the generated API docs, for cross-reference purposes. We're not *publishing* the HEAD support docs here.